CMakeLists.txt

```diff
@@ -69,7 +69,7 @@ option (PNET_OPTION_SNMP "" OFF)
 option (PNET_USE_ATOMICS "Enable use of atomic operations (stdatomic.h)" OFF)
 
 set(PNET_MAX_AR                 2
```

Comment:
Should the default value be 2? Seems like it would always generate an error (assuming that is what is being checked in pf_cmrpc.c)?

Reply:
Added a comment to describe the need for 2 AR instances in memory.
Done